docs/reference/adventureworks/armTemplates/auxiliary/policies.json

```diff
@@ -18903,6 +18903,555 @@
+            "policyRule": {
+              "if": {
+                "allOf": [
+                  {
+                    "field": "type",
+                    "equals": "Microsoft.MachineLearningServices/workspaces"
+                  },
+                  {
+                    "anyOf": [
+                      {
+                        "field": "Microsoft.MachineLearningServices/workspaces/hbiWorkspace",
+                        "exists": false
+                      },
+                      {
+                        "field": "Microsoft.MachineLearningServices/workspaces/hbiWorkspace",
+                        "notEquals": true
+                      }
+                    ]
+                  }
+                ]
+              },
+              "then": {
+                "effect": "[[parameters('effect')]"
+              }
+            }
+          },
+          "name": "Deny-MachineLearning-HbiWorkspace"
+        },
+        {
+          "properties": {
+            "displayName": "Deny-MachineLearning-PublicAccessWhenBehindVnet",
+            "mode": "Indexed",
+            "description": "Deny public access behind vnet for machine learning workspaces.",
+            "metadata": {
+              "version": "1.0.0",
+              "category": "Machine Learning"
+            },
+            "parameters": {
+              "effect": {
+                "type": "String",
+                "metadata": {
+                  "displayName": "Effect",
+                  "description": "Enable or disable the execution of the policy"
+                },
+                "allowedValues": [
+                  "Audit",
+                  "Disabled",
+                  "Deny"
+                ],
+                "defaultValue": "Deny"
+              }
+            },
+            "policyRule": {
+              "if": {
+                "allOf": [
+                  {
+                    "field": "type",
+                    "equals": "Microsoft.MachineLearningServices/workspaces"
+                  },
+                  {
+                    "anyOf": [
+                      {
+                        "field": "Microsoft.MachineLearningServices/workspaces/allowPublicAccessWhenBehindVnet",
+                        "exists": false
+                      },
+                      {
+                        "field": "Microsoft.MachineLearningServices/workspaces/allowPublicAccessWhenBehindVnet",
+                        "notEquals": false
+                      }
+                    ]
+                  }
+                ]
+              },
+              "then": {
+                "effect": "[[parameters('effect')]"
+              }
+            }
+          },
+          "name": "Deny-MachineLearning-PublicAccessWhenBehindVnet"
+        },
+        {
+          "properties": {
+            "displayName": "Deny-MachineLearningAks",
+            "mode": "Indexed",
+            "description": "Deny AKS creation (not attaching) in machine learning.",
```

Comment:
The description is not clear to me. What do we mean by creation (not attaching)? can we make it more clear?